services/config/config.sample.json

```diff
@@ -1,16 +1,16 @@
 {
     "postgres": {
-        "user": "cassandra",
-        "password": "cassandra",
-        "host": "localhost",
+        "user": "root",
+        "password": "5qvkFhOm1g97FoPbMoJyc8qY87m01ljiOMzv",
+        "host": "zl6cimmnex2.svc.trove.eqiad1.wikimedia.cloud",
```

Comment:
you are commiting secrets to the repo, this file shouldn't change (it is an example file of what should be there), please remove the commit from the history so those credentials won't be available